models/School.js

```diff
@@ -38,51 +38,65 @@ const SchoolSchema = new Schema({
             required: true
         }
     },
- 
+
     phone: {
         type: String,
-        required:[true, 'Phone number field is required']
+        required: [true, 'Phone number field is required']
     },
- 
-    email:{
+
+    email: {
         type: String,
-        required:[true, 'Email is required']
+        required: [true, 'Email is required']
     },
- 
+
     rating: {
         type: Number,
-        required:false
+        required: false
     },
- 
-    types:{
+
+    types: {
         type: String,
-        enum : ['Public','Private'],
-        required:[true, 'Types is required']
+        enum: ['Public', 'Private'],
+        required: [true, 'Types is required']
     },
- 
-    areas:{
+
+    areas: {
         type: [String],
-         enum : ['General','Technical','Vocational','Art Secondary Education' ],
-        required:[true, 'Areas is required']
+        enum: ['General', 'Technical', 'Vocational', 'Art Secondary Education'],
+        required: [true, 'Areas is required']
     },
-    network:{
+    network: {
         type: String,
-         enum : ['GO Network','Catholic Network','Municipality Schools','Private schools' ],
-        required:[true, 'Network is required']
+        enum: ['GO Network', 'Catholic Network', 'Municipality Schools', 'Private schools'],
+        required: [true, 'Network is required']
     },
     languageClasses: {
-       type: Boolean,
-       required:[true, 'Language classes is required']
-   } ,
-    comments: [CommentsSchema],
-   photo: {
-       type: String,
-       required:[true, 'Photo is required']
-   },
- 
-   website: {
-       type: String,
-       required:[true, 'Website is required']
+        type: Boolean,
+        required: [true, 'Language classes is required']
+    },
+
+    comments: [{
+        userid: { type: String },
```

Comment:
@pavelbidenko2018 , since we have CommentsSchema, would be good to replace all requirements for the comments there and in School schema just put reference

Reply:
So should we store the Comments separately?

Reply:
we have a Schema for comments in our project, so its kind a weird to have both: in school schema and separated Schema, choose please one 🙂

Reply:
@IrynaSpyrydonova Hi.
Structure fixed

Reply:
@pavelbidenko2018  Hi, thank you.